src/main/java/frc/trigon/robot/subsystems/sideshooter/SideShooterConstants.java

```diff
@@ -0,0 +1,54 @@
+package frc.trigon.robot.subsystems.sideshooter;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+public class SideShooterConstants {
+    private static final int SHOOTING_MOTOR_ID = 0;
+    private static final int ANGEL_MOTOR_ID = 0;
+    private static final int ENCODER_ID = 0;
+    private static final NeutralModeValue SHOOTING_NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final CANSparkMax.IdleMode ANGLE_NEUTRAL_MODE_VALUE = CANSparkMax.IdleMode.kBrake;
```

Comment:
This is not a neutral mode value

Reply:
why not?
even the name of that it's "NeutralModeValue"

Reply:
only in the CANSparkMax right?